Sources/BrowserServicesKit/ContentScopeScript/ContentScopeUserScript.swift

```diff
@@ -125,7 +125,7 @@ public final class ContentScopeUserScript: NSObject, UserScript, UserScriptMessa
                 isIsolated: Bool = false
     ) {
         self.isIsolated = isIsolated
-        let contextName = self.isIsolated ? "contentScopeScriptsIsolated" : "contentScopeScripts";
+        let contextName = self.isIsolated ? "contentScopeScriptsIsolated" : "contentScopeScripts"
```

Comment:
Oh wow, long time since I've seen one of them. I forgot it was even an option in swift!